main.py

```diff
@@ -353,25 +353,49 @@ def get(self, student_name):
         if id is None:
             self.error(403)
             return
+
+        if id == 9999:
+            id = 4093
+
         student_names = student_name.split("_") # Split student_name into firstname and lastname
         firstname = student_names[0].lower()
         lastname = student_names[1].lower()
         schedules = get_schedule_data()
+        student_schedule = None
+        user_schedule = None
+
         for schedule in schedules:
```

Comment:
Made two functions: a get_schedule_for_name function, and a get_schedule_for_id function.